package.json

```diff
@@ -47,7 +47,7 @@
     "on-finished": "~2.3.0",
     "parseurl": "~1.3.3",
     "path-to-regexp": "0.1.7",
-    "proxy-addr": "~2.0.5",
+    "proxy-addr": "2.0.6",
```

Comment:
Be sure to keep the `~` symbol; if a 2.0.7 is published, then a user upgrading from a 4.17 version of a 4.18 version would end up downgrading the `proxy-addr` installed.